### PR TITLE
Cleanup GetRaftCluster() and fix API errors.

### PR DESCRIPTION
**Pull request by huachaohuang**
1. `GetRaftCluster()` actually never return error.
2. Errors should not be rendered into JSON directly, that will output nothing...

### CODE REVIEW

**Comment by huachaohuang** [comment]
@siddontang @overvenus

**Comment by overvenus** [comment]
LGTM

File: server/api/balancer.go [hunk 1]
```diff
@@ -38,13 +38,9 @@ func newBalancerHandler(svr *server.Server, rd *render.Render) *balancerHandler
...
 func (h *balancerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	cluster, err := h.svr.GetRaftCluster()
-	if err != nil {
-		h.rd.JSON(w, http.StatusInternalServerError, err)
-		return
-	}
+	cluster := h.svr.GetRaftCluster()
 	if cluster == nil {
-		h.rd.JSON(w, http.StatusOK, nil)
+		h.rd.JSON(w, http.StatusInternalServerError, errNotBootstrapped.Error())
```
Comment (qiuyesuifeng) [inline server/api/balancer.go]: Why not directly using `errNotBootstrapped` but `errNotBootstrapped.Error()`?

File: server/api/event.go [hunk 2]
```diff
@@ -133,11 +125,7 @@ func (h *wsHandler) fetchEventFeed() {
 	for {
 		time.Sleep(1 * time.Second)
 
-		cluster, err := h.svr.GetRaftCluster()
-		if err != nil {
-			log.Error(err)
-			continue
-		}
+		cluster := h.svr.GetRaftCluster()
```
Comment (qiuyesuifeng) [inline server/api/event.go]: line 126, we can use `time.Sleep(time.Second)`.

**Comment by qiuyesuifeng** [comment]
LGTM
But the CI failed :(

**Pull request merged**